nixos/modules/services/search/elasticsearch.nix

```
@@ -149,6 +157,18 @@ in
       default = true;
     };
 
+    user = mkOption {
+      type = types.str;
+      default = "elasticsearch";
+      description = "User account under which Elasticsearch runs.";
```

```suggestion
      description = ''
        User account under which Elasticsearch runs.

        If left as the default value this user will automatically be created on system activation, otherwise you are responsible for ensuring the user exists before the Elasticsearch service starts.
      '';
```

what about this? and something like that for `group` too?